### DISCUSSION TITLE
Potential issues related to logarithm function

### DISCUSSION THREAD

**Discussion by th1275** [yalmip/YALMIP#1032] [Possible bug or missing functionality]
Hi all,

I developed a optimization model in which the objective function contains logarithm function as barrier. The program runs well in old revision of Yalmip. However, I just switch to the latest version today, and the error massage pop up saying the "log" operator cannot be created. I am not sure if it is the problem of latest version or from my end (but I switch back to the old version, it works again). Is there any one else seeing the similar problem?

Thanks

**Comment by johanlofberg** [comment]
You would have to supply a reproducible example as log should work (although my guess is that you haven't set up the path correctly after installing the new version)

**Reply by th1275** [reply]
Thanks for your quick reply. I will look deeper into the issues and try to prepare a reproducible example. Just for some information, I run "yalmiptest" every time when I switch to a different version and there is no error reported from the "yalmiptest".
